hoomd/error.py

```diff
@@ -37,6 +37,19 @@ class TypeConversionError(ValueError):
     pass
 
 
+class GPUNotAvailableError(NotImplementedError):
+    """Error for when a GPU specific feature was requested without a GPU."""
+    pass
+
+
+class _NoGPU:
```

Comment:
Should you remove this class from `hoomd/util.py`?

Reply:
You are right, thanks!